python/lsst/ts/ATWhiteLightSource/wlsCSC.py

```diff
@@ -59,12 +61,26 @@ class WhiteLightSourceCSC(salobj.ConfigurableCsc):
     hardware_listener_interval : int/float
         frequency, in seconds, that we check in on the hardware
     """
-    def __init__(self, config_dir=None, initial_state=salobj.State.STANDBY, initial_simulation_mode=0):
-        schema_path = pathlib.Path(__file__).resolve().parents[4].joinpath("schema", "ATWhiteLight.yaml")
-        super().__init__("ATWhiteLight", index=0, schema_path=schema_path, config_dir=config_dir,
-                         initial_state=initial_state, initial_simulation_mode=initial_simulation_mode)
+
+    def __init__(
+        self, config_dir=None, initial_state=salobj.State.STANDBY, simulation_mode=0
+    ):
+        schema_path = (
+            pathlib.Path(__file__)
+            .resolve()
+            .parents[4]
+            .joinpath("schema", "ATWhiteLight.yaml")
+        )
+        super().__init__(
+            "ATWhiteLight",
+            index=0,
+            schema_path=schema_path,
+            config_dir=config_dir,
+            initial_state=initial_state,
+            simulation_mode=simulation_mode,
+        )
         self.kiloarcModel = None
-        
+
         self.detailed_state = WLSDetailedState.OFFLINE
```

Comment:
if you decide to publish this as an event, you could rename `self.detailed_state` -> `self. kilo_arc_state` and make this a `@property` with a `@kilo_arc_state.setter` that will  handle the event publishing.